TensorLib/NumpyRepr.lean

```diff
@@ -514,5 +526,57 @@ def parseFile (path: System.FilePath) : IO (Except String NumpyRepr) := do
 
 end Parse
 
+section Save
+
+private def pushList (a : ByteArray) (xs : List UInt8) : ByteArray := xs.foldl (fun a b => a.push b) a
```

Comment:
Is it more efficient to use ByteArray.append, List.toArray ?

Reply:
Probably.  Will swap.